app/gist/controller.js

```diff
@@ -68,16 +100,21 @@ export default Ember.Controller.extend({
   /**
    * Set the initial file columns
    */
-  initializeColumns: observer('model', function() {
-    var files = this.get('model.files');
+  initializeColumns: observer('model', 'numColumns', function() {
```

Comment:
Might be nice to refactor this to use a computed property rather than an observer